src/functions/makeMosfet.ts

```diff
@@ -15,30 +18,58 @@ export const makeAngleSlider = (centerX: number, centerY: number, radius: number
     startAngle: startAngle,
     endAngle: endAngle,
     CCW: CCW,
-    minValue: minValue,
-    maxValue: maxValue,
-    value: minValue,
+    minValue: minValue, // TODO: rename this to voltageDifference or something
+    maxValue: maxValue, // TODO: rename this to voltageDifference or something
```

Comment:
if named the same, do not need to add key and value as it will infer this itself